### DISCUSSION TITLE
rspamd on cyberpanel

### DISCUSSION THREAD

**Discussion by linux-mafia** [rspamd/rspamd#4476] [Q&A]
Hi team, 
Can you tell me procedure to set up rspamd externally into cyberpanel?

the built in rspamd is being charged in cyberpanel and no gui access as well. 

how to add external spam filter like rspamd on cyberpanel. 

Regards

**Comment by caygri** [comment]
Hello, 

I found this https://community.cyberpanel.net/t/how-to-install-and-configure-rspamd-on-cyberpanel/36074
or 

https://cyberpanel.net/blog/rspamd-gui/

Both tutorial is on official cyberpanel site.

**Comment by linux-mafia** [comment]
hi elena.
yeah i was looking for gui feature or smthing like mailwatch where i could
see outgoing ingoing and spams and  trackers.
like spam from cfg exploit scanner and more

also installing it externally at cpanel. not from inside
